props/src/main/java/org/motechproject/nms/props/domain/FinalCallStatus.java

```diff
@@ -19,13 +22,13 @@ public int getValue() {
     }
 
     public static boolean isValidEnumValue(int i) {
-        return (i == 1 || i ==2 || i == 3);
+        return (i == SU || i == FA || i == RE);
     }
 
     public static FinalCallStatus fromInt(int i) {
-        if (i == 1) { return SUCCESS; }
-        if (i == 2) { return FAILED; }
-        if (i == 3) { return REJECTED; }
+        if (i == SU) { return SUCCESS; }
+        if (i == FA) { return FAILED; }
+        if (i == RE) { return REJECTED; }
```

Comment:
also, this seems like a standard performant way to get enum from int. http://stackoverflow.com/questions/11047756/getting-enum-associated-with-int-value

No need to change anything, just googling around on a friday and trying to write performance enhancing code.

Oh and switch-case :see_no_evil: is so much better than if, according to studies by me.

In other news, why did I go to college when I could have learnt everything from stack overflow in 4yrs?